app/src/models/reviewStorage.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+const db = require("../config/db");
+
+class ReviewStorage {
+  static getReviewInfo(movie_id) {
+    return new Promise((resolve, reject) => {
+      let query;
+      let params = [];
+
+      if (id) {
+        query = "SELECT * FROM review WHERE movie_id = ?";
+        params = [movie_id];
+      } else {
+        console("리뷰 조회에 movie_id가 필요합니다.");
```

Comment:
```suggestion
        console.log("리뷰 조회에 movie_id가 필요합니다.");
```
이 부분은 서비스에서 처리해야된다고 생각합니다